### PR TITLE
Per diem rates and departments specs

### PR DESCRIPTION
**Pull request by snarayanank2**
Per diem rate API specs

### CODE REVIEW

File: src/components/schemas/per_diem_rate.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    rate:
+      type: integer
+      example: 10.0
+      description: Amount per night
+
+
+per_diem_rate_in:
+  type: object
+  additionalProperties: False
+  required: ["name", "currency", "rate"]
```
Comment (aditya999123) [inline src/components/schemas/per_diem_rate.yaml:58]: marshmallow schema has is_enabled also as required
either we make it required here or put true as default in marshmallow (same as projects and categories)

File: reference/admin.yaml [hunk 2]
```diff
@@ -910,3 +968,92 @@ paths:
...
+  /admin/per_diem_rates:
+    get:
+      tags:
+        - Per Diem Rates
+      summary: List per diem rates
+      operationId: per_diem_rates_list
+      parameters:
+        - $ref: '#/components/parameters/created_at'
+        - $ref: '#/components/parameters/updated_at'
+        - $ref: '#/components/parameters/display_name'
```
Comment (aditya999123) [inline reference/admin.yaml:980]: per_diem_rate does not have a view like projects/categories, thus querying can only be done on name

File: src/components/schemas/per_diem_rate.yaml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    updated_at:
+      $ref: ./fields.yaml#/updated_at
+    is_enabled:
+      $ref: ./fields.yaml#/is_enabled
+    name:
+      type: string
+      maxLength: 255
+      example: Engine
+      description: Currency name
+    description:
```
Comment (aditya999123) [inline src/components/schemas/per_diem_rate.yaml:20]: might have to specify nullable as true

File: src/components/schemas/per_diem_rate.yaml [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+      $ref: './fields.yaml#/id_integer'
+    is_enabled:
+      $ref: ./fields.yaml#/is_enabled
+    name:
+      type: string
+      maxLength: 255
+      example: Engine
+      description: Currency name
+    description:
+      $ref: ./fields.yaml#/description
```
Comment (aditya999123) [inline src/components/schemas/per_diem_rate.yaml:70]: might have to specify nullable as true

File: src/components/schemas/per_diem_rate.yaml (hunk 1 above)
Comment (snarayanank2) [reply src/components/schemas/per_diem_rate.yaml:58]: will add here

File: reference/admin.yaml (hunk 2 above)
Comment (snarayanank2) [reply reference/admin.yaml:980]: will fix - thanks

File: src/admin/openapi.yaml [hunk 5]
```diff
@@ -27,9 +27,16 @@ tags:
     description: >
       These APIs gives you access to manage your organisation's Cost Centers
       data.
+  - name: Per Diem Rates
+    description: |
+      These APIs gives you access to manage your organisation's Categories data.
```
Comment (jayachandhar) [inline src/admin/openapi.yaml:32]: Categories  -> Per Diem Rates

File: src/admin/openapi.yaml [hunk 6]
```diff
@@ -27,9 +27,16 @@ tags:
...
+  - name: Departments
+    description: |
+      These APIs gives you access to manage your organisation's Categories data.
```
Comment (jayachandhar) [inline src/admin/openapi.yaml:35]: Categories -> Departments

File: src/admin/paths/admin@per_diem_rates.yaml [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+            - data
+  responses:
+    '200':
+      description: OK
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              url:
```
Comment (jayachandhar) [inline src/admin/paths/admin@per_diem_rates.yaml:68]: url is not required

File: src/admin/paths/admin@departments.yaml [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+            - data
+  responses:
+    '200':
+      description: OK
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              url:
```
Comment (jayachandhar) [inline src/admin/paths/admin@departments.yaml:68]: url is not required

**Pull request merged**